assets/css/main.css

```diff
@@ -246,46 +246,71 @@ html, body {
     list-style: none;
 }
 
-/* index thumbnails */
-.course-thumbnails .advert-thumbnail,
-.course-thumbnails .course-thumbnail {
-    height: 330px;
+
+/* index frontpage */
+
+.frontpage-panel {
+    margin: 3%;
+    margin-top: 30px;
+}
+
+.frontpage-panel .panel-heading {
+    border-bottom: solid 1px #eee;
+    font-weight: 500;
+}
+
+.frontpage .card {
+    margin-bottom: 20px;
+    border: solid 1px #ccc;
     position: relative;
+    height: 355px;
+    border-radius: 4px;
+    padding: 4px;
 }
-.course-thumbnails .advert-thumbnail {
+
+.frontpage .advert-card {
     background-color: #d9edf7;
     border-color: #bce8f1;
 }
-.course-thumbnails .advert-thumbnail .image,
-.course-thumbnails .course-thumbnail .image {
-    height: 130px;
-    overflow: hidden;
-    text-align: center;
-}
-.course-thumbnails .course-thumbnail .image {
-    background: #eee;
+
+.frontpage .course-image {
```

Comment:
Should the image have a hover state? Maybe something like 5% lighter, as it's clickable, but currently now shown at all.

Reply:
Yes, a good idea, I will add this for now :)

Reply:
Fixed with clickable cards